apps/api/src/domain/services/query-services/playlist-query.service.ts

```diff
@@ -4,8 +4,8 @@ import { CommandInfoService } from '../../../app/controllers/command/services/co
 import { DomainModelCtor } from '../../../lib/types/DomainModelCtor';
 import { REPOSITORY_PROVIDER_TOKEN } from '../../../persistence/constants/persistenceConstants';
 import { PlaylistViewModel } from '../../../view-models/buildViewModelForResource/viewModels';
-import { AudioItem } from '../../models/audio-item/entities/audio-item.entity';
 import BaseDomainModel from '../../models/BaseDomainModel';
+import { AudioItem } from '../../models/audio-item/entities/audio-item.entity';
```

Comment:
Why did this move? Is your format imports working? It looks like the previous order was correct.

Reply:
Let's keep an eye on this issue.